config/rbac/role.yaml

```diff
@@ -29,3 +29,10 @@ rules:
   - get
   - patch
   - update
+- apiGroups:
+  - '*'
+  resources:
+  - '*'
+  verbs:
+  - list
+  - update
```

Comment:
I dont think we need update access for all apiGroups and resources.

Reply:
I think we do. Otherwise, we have to list all known GVKs. Also, for future CRDs, we will get the same error when we're trying to list/update them.

Reply:
i see update for OwnerRef injection.

Reply:
this is a generated file @nan-yu and will be overwritten

Reply:
I created a PR #174